pandas/tests/indexes/datetimes/test_timezones.py

```diff
@@ -434,18 +434,20 @@ def test_dti_tz_localize_utc_conversion(self, tz):
         with pytest.raises(pytz.NonExistentTimeError):
             rng.tz_localize(tz)
 
+    @pytest.mark.parametrize('ambiguous', [True, False])
+    @pytest.mark.parametrize('tz', [
+        'UTC', 'Europe/London', 'US/Eastern', 'Asia/Tokyo',
+        'dateutil/US/Pacific','dateutil/Asia/Singapore', tzutc(), tzlocal(),
+        pytz.FixedOffset(300), pytz.FixedOffset(0), pytz.FixedOffset(-300)
+    ])
     @pytest.mark.parametrize('idx', [
         date_range(start='2014-01-01', end='2014-12-31', freq='M'),
         date_range(start='2014-01-01', end='2014-12-31', freq='D'),
         date_range(start='2014-01-01', end='2014-03-01', freq='H'),
```

Comment:
if i extend this date range (`idx2`) to cover a start-of-DST transition, i get a `NonExistentTimeError`. I'd like to handle that case too before this goes in